client/signup/config/flows-pure.js

```diff
@@ -388,7 +388,6 @@ export function generateFlows( {
 				'courses',
 				'store-options',
 				'store-features',
-				'design-setup-site',
```

Comment:
Removing the design setup step was the only way I could get the store redirection to work, but I realize now that would break the site setup flow for everyone else (thankfully e2e tests caught that!) Instead of goToNextStep we need to find a way to skip design selection and end the flow without removing it for other flows.

Reverting in #60933